docs/cli.rst

```diff
@@ -211,6 +211,17 @@ The VHDL revision can be specified through the :ref:`python_interface`
 Alternatively, environment variable ``VUNIT_VHDL_STANDARD`` can be set to
 ``1993``, ``2002``, ``2008`` or ``2019``.
 
+.. note:: ``93``, ``02``, ``08`` or ``19`` can be also used.
```

Comment:
I would write this in the previous paragraph:

```
Alternatively, environment variable ``VUNIT_VHDL_STANDARD`` can be set to
``93``|``1993``, ``02``|``2002``, ``08``|``2008`` (default) or ``19``|``2019``.
```

Reply:
Looks great!